campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -40,7 +40,7 @@
 
   @blockable: checkbox - Check to allow user to block this snippet.
   @block_button_text - small text - Text to display while hovering over opt out button. Default "Remove this"
-
+  @do_not_autoblock - Check to prevent auto-blocking when user takes action.
```

Comment:
Used inverted / negative variable name to set the default to False and also match this in the admin interface.